acstools/polarization_tools.py

```diff
@@ -0,0 +1,205 @@
+            Photometric measurement in POL0 filter. Units: electrons or electrons/second.
+        pol60 : float
+            Photometric measurement in POL60 filter. Units: electrons or electrons/second.
+        pol120 : float
+            Photometric measurement in POL120 filter. Units: electrons or electrons/second.
+        filter_name : str
+            Name of the filter crossed with the polarization filter, e.g., F606W.
+        detector : str
+            Name of the ACS detector used for the observation. Must be either WFC or HRC.
+        pav3 : float
+            Position angle of the HST V3 axis. This is stored in the ACS primary header under
+            keyword PA_V3. Units: degrees.
+        """
+
+        self.pol0 = pol0
+        self.pol60 = pol60
+        self.pol120 = pol120
+
+        self.filter_name = filter_name
+        self.detector = detector
+        self.pav3 = pav3
+
+        # Check if detector is a valid value.
+        if self.detector.lower() not in ['wfc', 'hrc']:
+            raise ValueError('Detector must be either WFC or HRC')
+
+        self.stokes_i = None
+        self.stokes_q = None
+        self.stokes_u = None
+
+        self.polarization = None
+        self.angle = None
+
+        # Get correction terms that we need from the polarization tables.
+        tables = PolarizerTables()
+        leak_tab = Table(tables.table_data['transmission'][detector.lower()])
+        self.transmission_correction = leak_tab[np.where(leak_tab['filter'] == filter_name.upper())]['correction'][0]
+
+        eff_tab = Table(tables.table_data['efficiency'][detector.lower()])
+        self.c0 = eff_tab[np.where(eff_tab['filter'] == filter_name.upper())]['pol0'][0]
+        self.c60 = eff_tab[np.where(eff_tab['filter'] == filter_name.upper())]['pol60'][0]
+        self.c120 = eff_tab[np.where(eff_tab['filter'] == filter_name.upper())]['pol120'][0]
+
+    def calc_stokes(self):
+        """
+        Calculate Stokes parameters using attributes set at initialization.
+        """
+
+        self.stokes_i, self.stokes_q, self.stokes_u = self._calc_stokes([self.pol0, self.pol60, self.pol120],
+                                                                        [self.c0, self.c60, self.c120])
+
+    def _calc_stokes(self, pol_measurements, correction_terms):
+        """
+        Method for determining Stokes parameters for ACS observations.
+
+        Parameters
+        ----------
+        pol_measurements : list or tuple
+            Measurements in each of the three ACS polarization filters. Must be in order
+            of [POL0, POL60, POL120]. Units: electrons or electrons/second.
+        correction_terms : list or tuple
+            Polarizer efficiency correction values. Must be in order of [POL0, POL60, POL120].
+            These can be found in the tables provided by `acstools.acstools.polarization_tools.PolarizerTables()`
+            or in Table 5.6 in Section 5.3 of the ACS Data Handbook.
+
+        Returns
```

Comment:
Hidden method's docstring is not rendered, so you need to copy the "Returns" info here to `calc_stokes` also if you decide to keep this as a separate hidden method.